mautic/mautic
# Add patch for UnitOfWork warnings on PHP 7.3

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | X
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Fixes #7843 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Checkout the `staging` branch locally, no need to install Mautic. Just run `composer install`
2. Run `php build/package_release.php`
3. Open `build/packaging/vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php` on line 2636: it should say `continue`. Same for line 2665.

#### Steps to test this PR:
1. Checkout this PR locally (`git checkout fix-7843`)
2. Run `rm -rf vendor` just to be sure we have a clean state
3. Run `composer install`
4. Run `php build/package_release.php`
5. Open `build/packaging/vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php` on line 2636: it should say `break`. Same for line 2665.

#### List deprecations along with the new alternative:
1. n/a
2. 

#### List backwards compatibility breaks:
1. n/a - we're already on the latest 2.5 version of doctrine/ORM and it doesn't receive updates anymore (due to it being end of life), there won't be side effects for the 2.x branch of Mautic as we won't be updating Doctrine ORM to 2.6 or higher in this version anymore.
2.
